models/stdp_pl_connection_hom.cpp

```diff
@@ -66,7 +66,14 @@ STDPPLHomCommonProperties::set_status( const DictionaryDatum& d,
   CommonSynapseProperties::set_status( d, cm );
 
   updateValue< double >( d, names::tau_plus, tau_plus_ );
-  tau_plus_inv_ = 1. / tau_plus_;
+  if ( tau_plus_ > 0. )
```

Comment:
Is there a constraint on how small tau_plus is? If there wasn't before, then you don't need to add it, but it's probably worth documenting if there's a numeric instability here.